draft-ietf-quic-transport.md

```diff
@@ -1338,9 +1338,29 @@ Servers MUST drop incoming packets under all other circumstances.
 ## Version Negotiation
 
 Version negotiation ensures that client and server agree to a QUIC version
-that is mutually supported. A server sends a Version Negotiation packet in
-response to each packet that might initiate a new connection, see
-{{packet-handling}} for details.
+that is mutually supported. The client's Initial packet contains two version
+indications:
+
+* A packet version which indicates which version it has been encoded
+  with. This SHOULD be a version which the client expects most servers
+  to support (most likely the oldest common version).
+  
+* A list of versions which the client supports in the
+  "supported_versions" transport parameter).
+
+If the server understands the packet version -- even if it is not its
+most preferred version -- then it deprotects the packet and selects
+the most preferred version out of the client's supported versions
+list, and responds with that version. This allows a client and server
```

Comment:
Aside from sentence construction here, which needs work, I don't understand how you believe this should work.  If you believe that a server can continue a connection, how can it continue from an Initial taken from a different version?